go.mod

```diff
@@ -171,4 +171,5 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.30.2
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.30.2
 	k8s.io/sample-controller => k8s.io/sample-controller v0.30.2
+	volcano.sh/apis => github.com/TaiPark/volcano-apis v1.10.0
```

Comment:
Please do not you your own repo

Reply:
I will change it when https://github.com/volcano-sh/apis/pull/132 merged

Reply:
Already change to https://github.com/volcano-sh/apis/pull/135